eth/v1alpha1/validator.proto

```diff
@@ -292,6 +295,26 @@ enum ValidatorStatus {
   EXITED = 6;
 }
 
+message ValidatorStatusMetadata {
```

Comment:
You can move this message inside of the `ValidatorStatusResponse` message, as it's only used in there

Reply:
Done!